lib/security/security-spec.js

```diff
@@ -0,0 +1,50 @@
+var assert = require('assert');
+var sampleUserProfileData = require('../../test/fixtures/sampleUserProfile.json');
+var aes = require('./aes.js');
+var sha256 = require('./sha256.js');
+var sampleSecurityData = require('../../test/fixtures/sampleSecurityData.json');
+
+describe("Test Data Encyption, Hashing and Decryption", function() {
+  it('should error when you try to hash a null value', function() {
+    var hash = sha256.hash(null);
+    assert.equal(true, (hash instanceof Error), 'Should return an error.');
```

Comment:
In this case `Error()` should be thrown and you need use `expect()`
expect(sha256.hash(null)).to.throw(new Error("Cannot Encrypt Profile Data - Data or Secret key is Null."));